sdks/go/pkg/beam/io/bigqueryio/bigquery.go

```diff
@@ -93,18 +93,37 @@ func Read(s beam.Scope, project, table string, t reflect.Type) beam.PCollection
 	return query(s, project, fmt.Sprintf("SELECT * from [%v]", table), t)
 }
 
+// QueryOptions represents additional options for executing a query.
+type QueryOptions struct {
+	// UseStandardSQL enables BigQuery's Standard SQL dialect when executing a query.
+	UseStandardSQL bool
+}
+
+// UseStandardSQL enables BigQuery's Standard SQL dialect when executing a query.
+func UseStandardSQL(qo *QueryOptions) error {
+	qo.UseStandardSQL = true
+	return nil
+}
```

Comment:
Part of the functional options is to have the user level functions return functions. This allows for closures over user configuration for more complicated options later on.  Probably doesn't matter in this case, but otherwise:

eg. 
```suggestion
func UseStandardSQL() func(qo *QueryOptions) error {
	return func(qo *QueryOptions) error
		qo.UseStandardSQL = true
		return nil
	}
}
```

I'd probably drop the "use" so it's just StandardSQL.  When there are more of these, the repeated Use prefix will get tiresome and reduce readability.

Reply:
Done.  Swapped to returning a function.

I left the `Use` prefix to match the the Python and Java BigQuery IO packages and base BQ library. Other config options, like flatten results and validation, don't follow this pattern so I expect this to be a one-off (feels like legacy decision from the BigQuery API design). Happy to change tho, LMK!

Reply:
Works for me! Consistency with the deeper BigQuery options certainly trumps local verbosity.

Thanks!